app/src/main/res/layout/item_person.xml

```diff
@@ -1,75 +1,72 @@
-        android:gravity="center"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/picture"
         tools:text="@string/about_julien"/>
 
-    <TextView
+    <ImageView
+        android:id="@+id/linkedIn"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:id="@+id/role"
-        android:textSize="@dimen/text_medium"
-        android:gravity="center"
-        tools:text="@string/about_julien_role"/>
+        android:layout_marginStart="8dp"
+        android:layout_marginTop="8dp"
+        android:background="?android:attr/selectableItemBackground"
+        android:contentDescription="@null"
+        android:padding="@dimen/padding_micro"
+        android:tint="@color/red"
+        app:layout_constraintEnd_toStartOf="@+id/email"
+        app:layout_constraintHorizontal_chainStyle="packed"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/name"
+        app:srcCompat="@drawable/ic_linkedin"/>
 
-    <LinearLayout
-        android:layout_width="match_parent"
-        android:layout_height="wrap_content"
-        android:orientation="horizontal"
-        android:gravity="center">
-
-        <ImageView
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:id="@+id/linkedIn"
-            app:srcCompat="@drawable/ic_linkedin"
-            android:tint="@color/red"
-            android:background="?android:attr/selectableItemBackground"
-            android:padding="@dimen/padding_micro"
-            android:contentDescription="@null"/>
-
-        <ImageView
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:id="@+id/email"
-            app:srcCompat="@drawable/ic_email"
-            android:tint="@color/red"
-            android:background="?android:attr/selectableItemBackground"
-            android:padding="@dimen/padding_micro"
-            android:contentDescription="@null"/>
-
-    </LinearLayout>
-
-    <TextView
+    <ImageView
+        android:id="@+id/email"
         android:layout_width="wrap_content"
         android:layout_height="wrap_content"
-        android:id="@+id/description"
-        android:textSize="@dimen/text_small"
-        android:textStyle="italic"
-        android:gravity="center"
-        tools:text="@string/about_julien_description"/>
-</LinearLayout>
\ No newline at end of file
+        android:layout_marginEnd="8dp"
+        android:layout_marginTop="8dp"
+        android:background="?android:attr/selectableItemBackground"
+        android:contentDescription="@null"
+        android:padding="@dimen/padding_micro"
+        android:tint="@color/red"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toEndOf="@+id/linkedIn"
```

Comment:
Lowercase `i` to remain consistent?

Reply:
It should actually all be `linkedIn` or `linked_in` if uppercases aren't available. So the String Id should change here, not this.